cpp/src/Ice/Proxy.cpp

```diff
@@ -394,7 +394,7 @@ Ice::ObjectPrx::_endpointSelection(EndpointSelectionType newType) const
 }
 
 ReferencePtr
-Ice::ObjectPrx::_endpoints(EndpointSeq newEndpoints) const
+Ice::ObjectPrx::_endpoints(EndpointSeq newEndpoints) const // NOLINT(performance-unnecessary-value-param)
 {
     vector<EndpointIPtr> endpoints;
```

Comment:
Not related to PR, but can't we do the comparison without creating this intermediary vector?

Reply:
We could, but then we need the intermediary vector in the usual path where we call changeEndpoints.